src/app/dashboard/plugin-config.html

```diff
@@ -1,13 +1,15 @@
-<md-dialog>
+<md-dialog flex="30">
   <md-dialog-content>
-    <!-- {{vm.plugin.title}} -->
+    <h2>{{vm.plugin.title}}</h2>
```

Comment:
What are your thoughts on adding the version as well? I think it would be good to have, especially in the sense of collaboration. Like if PersonA told PersonB to configure a certain field a specific way, but maybe PersonB was on an older version, it wouldn't be very intuitive that he's out of date and could potentially cause confusion.